### PR TITLE
Add `python -m` support

### PR DESCRIPTION
**Pull request by YogiLiu**
## Description
Running strawberry as a script, for example, start the debug server with the following command:

```shell
python -m strawberry server schema
```

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* #1475

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Running strawberry as a script, for example, start the debug server with the following command:

```shell
python -m strawberry server schema
```


---
Here's the preview release card for twitter:




Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to YogiLiu for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

**Review by patrick91 — APPROVED** [review]
Looks good to me!

image

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Release type: patch
+
+Running strawberry as a script, for example, start the debug server with the following command:
```
Comment (patrick91) [inline RELEASE.md:3]:
```suggestion
This release allows running strawberry as a script, for example, you can start the debug server with the following command:
```

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**